src/core/reactor.cc

```diff
@@ -2218,6 +2219,45 @@ void reactor::kill(pid_t pid, int sig) {
     ret.throw_if_error();
 }
 
+future<group_details> reactor::grp_detail(std::string_view groupname) noexcept {
+    return futurize_invoke([groupname, this] {
```

Comment:
Better to use coroutines. Most/all of the Seastar codebase doesn't use coroutines because until recently we supported C++17, but that's no longer the case and coroutines would be more readable here.

Reply:
For this issue, i was following the pattern already defined in the `src/core/reactor.cc`, could adopting coroutines be handled as a separate issue migrating all of the existing usage to the coroutines ?

Reply:
Hi @avikivity , does this patch `[scylladb-dev] [PATCH v1 00/11] co-routinize most of the CQL transport code` make this ticket: obsolete ? Thanks

Reply:
Got it, I picked this issue as it was labeled as a `good first issue` , this was a good beginner experience for me to clone the repo and get around tools. I would like to get more familiar with `coroutines` and interested in working on porting this code to coroutines. Thank you

Reply:
this comment is not resolved yet.